CMakeLists.txt

```diff
@@ -744,7 +753,7 @@ elseif(APPLE)
   set_target_properties(lovr PROPERTIES
     MACOSX_RPATH TRUE
     BUILD_WITH_INSTALL_RPATH TRUE
-    INSTALL_RPATH "@executable_path"
+    INSTALL_RPATH "@executable_path;@executable_path/../lib"
```

Comment:
Why was this added to rpath?

Reply:
if `lovr` is installed to `/usr/local/bin` then libraries will be in `/usr/local/lib`, etc

Reply:
Installing to a system-wide location like this isn't supported right now

Reply:
it does work though